app/models/submission.rb

```diff
@@ -145,6 +145,7 @@ def save_additional_form_fields(params)
       if assessment.is_section_dependant
         form_hash["section"] = (assessment.lecture?) ? course_user_datum.lecture : course_user_datum.section
       end
+      form_hash["ip"] = self.submitter_ip
```

Comment:
This change is part of a separate change that has been merged - it should not be included here